segregation/spatial_indexes.py

```diff
@@ -1886,8 +1903,33 @@ def _absolute_centralization(data, group_pop_var, total_pop_var):
     c_lons = np.array(data.centroid.x)
     c_lats = np.array(data.centroid.y)
     
-    center_lon = c_lons.mean()
-    center_lat = c_lats.mean()
+    if isinstance(center, str):
+        
+        if (center == "mean"):
+            center_lon = c_lons.mean()
+            center_lat = c_lats.mean()
+    
+        if (center == "median"):
+            center_lon = np.median(c_lons)
+            center_lat = np.median(c_lats)
+            
+        if (center == "population_weighted_mean"):
+            center_lon = np.average(c_lons, weights = t)
+            center_lat = np.average(c_lats, weights = t)
+    
+        if (center == "largest_population"):
+            center_lon = c_lons[np.where(t == t.max())].mean()
+            center_lat = c_lats[np.where(t == t.max())].mean()
+
+    if isinstance(center, tuple):
```

Comment:
`tuple` can be generalized to include `list` and `array` as well? as long as there are two numbers. Sth like this: 
```python
if np.array(center).shape==(2,):
```
Just a suggestion

Reply:
Good catch! I resolved this in https://github.com/pysal/segregation/pull/25/commits/3ae9657ae192624657a51fbf1e3ffc820ea5bb35

Also, the print errors are very informative if the user eventually passes an unexpected input.